controller/main.go

```diff
@@ -29,8 +28,10 @@ import (
 	"go.universe.tf/metallb/internal/logging"
 	"go.universe.tf/metallb/internal/version"
 
-	"github.com/go-kit/kit/log"
 	v1 "k8s.io/api/core/v1"
+
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
```

Comment:
More nitpick, but I would not move `"github.com/go-kit/kit/log"`, just put `"github.com/go-kit/kit/log/level"` after it.
the less diff you have the easier it is for people to rebase on top